packages/tupaia-web/src/layout/Sidebar/Sidebar.tsx

```diff
@@ -5,41 +5,122 @@
 import { TRANSPARENT_BLACK } from '../../constants';
 import { ExpandButton } from './ExpandButton';
+import { Photo } from './Photo';
+import { Breadcrumbs } from './Breadcrumbs';
+import { StaticMap } from './StaticMap';
+import { useEntity } from '../../api/queries';
 
 const MAX_SIDEBAR_EXPANDED_WIDTH = 1000;
-const MAX_SIDEBAR_COLLAPSED_WIDTH = 350;
+const MAX_SIDEBAR_COLLAPSED_WIDTH = 500;
+const MIN_SIDEBAR_WIDTH = 335;
+const PANEL_GREY = '#4a4b55';
 
 const Panel = styled.div<{
   $isExpanded: boolean;
 }>`
-  display: flex;
-  align-items: stretch;
-  flex-direction: column;
-  align-content: stretch;
   position: relative;
-  overflow: visible;
   background-color: ${TRANSPARENT_BLACK};
-  height: 100%;
-  cursor: auto;
   transition: width 0.5s ease, max-width 0.5s ease;
-  width: ${({ $isExpanded }) => ($isExpanded ? 45 : 30)}%;
-  min-width: 335px;
+  width: ${({ $isExpanded }) => ($isExpanded ? 55 : 30)}%;
+  min-width: ${MIN_SIDEBAR_WIDTH}px;
   max-width: ${({ $isExpanded }) =>
     $isExpanded ? MAX_SIDEBAR_EXPANDED_WIDTH : MAX_SIDEBAR_COLLAPSED_WIDTH}px;
+  overflow: visible;
+`;
+
+const ScrollBody = styled.div`
+  position: relative;
+  height: 100%;
+  overflow: auto;
+`;
+
+const TitleBar = styled.div`
+  position: sticky;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  background-color: ${TRANSPARENT_BLACK};
+  z-index: 1;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.2);
+`;
+
+const ExportButton = styled(Button).attrs({
+  variant: 'outlined',
+})`
+  font-size: 0.6875rem;
+`;
+
+const Title = styled(Typography)`
+  color: white;
+`;
+
+const Dropdown = styled.div`
+  background: ${PANEL_GREY};
+  width: 100%;
+  padding: 1rem;
+  font-size: 1rem;
+`;
+
+const ChartsContainer = styled.div<{
+  $isExpanded: boolean;
+}>`
+  display: grid;
+  background-color: ${PANEL_GREY};
+  grid-template-columns: repeat(auto-fill, minmax(${MIN_SIDEBAR_WIDTH}px, auto));
+  column-gap: 0.5rem;
+  row-gap: 0.5rem;
+  padding: ${({ $isExpanded }) => ($isExpanded ? '0 0.5rem 0.5rem' : '0 0 0.5rem 0')};
+`;
+
+const Chart = styled.div`
+  position: relative;
+  background-color: ${TRANSPARENT_BLACK};
+  // Use padding to maintain aspect ratio
+  padding-bottom: 80%;
 `;
 
 export const Sidebar = () => {
   const [isExpanded, setIsExpanded] = useState(false);
-
+  const { data: entityData } = useEntity('fiji');
+  const bounds = entityData?.location?.bounds;
   const toggleExpanded = () => {
     setIsExpanded(!isExpanded);
   };
 
   return (
     <Panel $isExpanded={isExpanded}>
       <ExpandButton setIsExpanded={toggleExpanded} isExpanded={isExpanded} />
+      <ScrollBody>
+        <Breadcrumbs />
+        {bounds ? (
+          <StaticMap polygonBounds={bounds} />
+        ) : (
+          <Photo title={entityData?.name} photoUrl={entityData?.photoUrl} />
+        )}
+        <TitleBar>
+          <Title variant="h5">Northern</Title>
```

Comment:
Wondering if this should be an `h3` or `h4` since `h5` skips a hierarchy level I think?

Reply:
yes good point